DirectProgramming/DPC++FPGA/ReferenceDesigns/gzip/src/gzipkernel_ll.cpp

```diff
@@ -75,13 +75,13 @@
 
 #include "gzipkernel_ll.hpp"
 #include "kernels.hpp"
-#include "pipe_array.hpp"
+#include "pipe_utils.hpp"
```

Comment:
Please add a comment to every design where you #include pipe_utils to indicate where the file is located.

Reply:
What is your preference for the relative directory to use here? Is this path clear?

DirectProgramming/DPC++FPGA/include/

Reply:
Added the following to the include lines:
#include "pipe_utils.hpp" // Included from DirectProgramming/DPC++FPGA/include/